apps/demos/Demos/Chat/Customization/jQuery/index.js

```diff
@@ -0,0 +1,70 @@
+$(async () => {
```

Comment:
why is it async?
```suggestion
$(() => {
```

Reply:
[a788963](https://github.com/DevExpress/DevExtreme/pull/28441/commits/a7889633ac3296ccaa25a0589dccff72c053b179)